reference/api/tasks.md

```diff
@@ -14,19 +160,19 @@ List all tasks globally, regardless of index. The `task` objects are contained i
 
 Tasks are always returned in descending order of `uid`. This means that by default, **the most recently created `task` objects appear first**.
 
-Task results are paginated and can be filtered. To learn more, refer to our [asynchronous operations](/learn/advanced/asynchronous_operations.md#filtering-tasks) guide.
+Task results are [paginated](#paginating-tasks) and can be [filtered](#filtering-tasks).
 
-#### Query parameters
+### Query parameters
 
-| Query Parameter | Description                                                                                                    |         Default Value          |
-|-----------------|----------------------------------------------------------------------------------------------------------------|:------------------------------:|
-| **limit**       | number of tasks to return                                                                                      |               20               |
-| **from**        | `uid` of the first task returned                                                                               | `uid` of the last created task |
-| **status**      | [filter tasks](/learn/advanced/asynchronous_operations.md#filtering-tasks) by their `status`                   |          all statuses          |
-| **type**        | [filter tasks](/learn/advanced/asynchronous_operations.md#filtering-tasks) by their `type`                     |           all types            |
-| **indexUid**    | [filter tasks](/learn/advanced/asynchronous_operations.md#filtering-tasks) by their `indexUid`. Case-sensitive |          all indexes           |
+| Query Parameter | Description                                                          |         Default Value          |
+|-----------------|----------------------------------------------------------------------|:------------------------------:|
+| **limit**       | number of tasks to return                                            |               20               |
+| **from**        | `uid` of the first task returned                                     | `uid` of the last created task |
+| **status**      | [filter tasks](#filtering-tasks) by their `status`                   |          all statuses          |
+| **type**        | [filter tasks](#filtering-tasks) by their `type`                     |           all types            |
+| **indexUid**    | [filter tasks](#filtering-tasks) by their `indexUid`. Case-sensitive |          all indexes           |
 
-### Example
+#### Example
```

Comment:
Can we define when example should be an H3 and when it should be an H4?

H4 in the settings API reference, H3 everywhere else?

Reply:
Wouldn't it be better to set it to H4 everywhere?

Reply:
That makes sense, but the actual goal should be a bit more comprehensive, imho. It's not necessarily establishing a specific h# for code samples, but creating a document structure in which that the desired the h# is the most sensible choice, if that makes sense.

Reply:
I like when Example and Response are both H4s, they look better
Plus, if we ever use `sidebarDepth`, they won't show up in the sidebar
<img width="1067" alt="Screen Shot 2022-08-04 at 19 26 52" src="https://user-images.githubusercontent.com/90181761/182886328-f8ec1c8d-84bd-4a8a-a09d-8603d2b40c20.png">

Reply:
I agree @maryamsulemani97 , they do look better as H4s. However I'm a bit concerned about parent-child relationships. With the current structure, this would make example and response children of whichever H3 came before—most likely `Query parameters`, `Body`, or similar.

One solution would be to move the example and response earlier, to immediately follow the endpoint description. That way they would be children of the endpoint H2—however we would be jumping from an H2 to an H4. Does that matter to us? 🤔

Reply:
>However I'm a bit concerned about parent-child relationships. With the current structure, this would make example and response children of whichever H3 came before—most likely Query parameters, Body, or similar.

I did not think of this. We should keep Example as an H3, I guess.